tests/Maker/MakeMigrationTest.php

```diff
@@ -113,7 +113,7 @@ public function getTestDetails(): \Generator
 
                 $this->assertStringContainsString('You have 1 available migrations to execute', $output);
                 $this->assertStringContainsString('Success', $output);
-                $this->assertCount(14, explode("\n", $output), 'Asserting that very specific output is shown - some should be hidden');
+                $this->assertCount(17, explode("\n", $output), 'Asserting that very specific output is shown - some should be hidden');
```

Comment:
We've added additional lines to the output. This test should be changed in a separate PR to assert exactly what we're looking for rather than total number of lines. - Todo added on a post-it

Reply:
https://github.com/symfony/maker-bundle/pull/1254